### DISCUSSION TITLE
是不是最近那个了，刷新后0可用, CF也似乎要挂了

### DISCUSSION THREAD

**Discussion by lanseria** [vvbbnn00/WARP-Clash-API#218] [General]
➜ sudo docker compose exec warp-clash python3 app.py optimize
[sudo] password for bmc-r1:
WARN[0000] /home/bmc-r1/share/code/WARP-Clash-API/docker-compose.yaml: `version` is obsolete
Getting IPv4 entrypoints
CloudflareWarpSpeedTest

1000 / 1000 [-------------------------------------------------------------------------------------------] Available: 0

[Info] The total number of IP addresses in the complete speed test results is 0, so skipping the output.
mv: can't rename 'result.csv': No such file or directory
Getting IPv6 entrypoints
CloudflareWarpSpeedTest

1000 / 1000 [-------------------------------------------------------------------------------------------] Available: 0

[Info] The total number of IP addresses in the complete speed test results is 0, so skipping the output.
mv: can't rename 'result.csv': No such file or directory

**Comment by hahach** [comment]
有可能，我这家庭宽带还有几个能用，移动端全挂

**Reply by lanseria** [reply]
用ipv6可行

**Reply by lanseria** [reply]
ipv6也全军覆没

**Comment by lanseria** [comment]
最新 以后可能 `cloudflare` 也要打不开了，刚刚似死似活的

**Comment by naweiduxian** [comment]
我这扫ip倒是能扫到，但是怎么现在扫出来没有端口了？
